pie chart updates (#219)
patternfly/training-scenarios: react-charts/pie-chart/intro.md

PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly.

> PatternFly provides a collection of React-based components you can use to build patterns with consistent markup, styling, and behavior.

Reviewer: ```suggestion
PatternFly provides a collection of React-based components that are used to build patterns with consistent markup, styling, and behavior.
```

> PatternFly React charts are composed using separate components, so features are more easily customizable. This course covers PatternFly pie charts - starting with a simple chart, tooltips, a legend, and concluding by changing the theme color.

Reviewer: ```suggestion
PatternFly React charts are composed using separate components, so features are more easily customizable. This course covers PatternFly pie charts starting with a simple chart, tooltips, a legend, and concluding by changing the theme color.
```
